src/custom-elements.js

```diff
@@ -451,6 +459,9 @@ let Deferred;
         const walker = createTreeWalker(root);
         do {
           const node = /** @type {!HTMLElement} */ (walker.currentNode);
+          if (node.getAttribute('is')) {
```

Comment:
I agree that not looking at the `is` attribute would be ideal, but like you pointed out I don't think it's possible to avoid it for parsed elements. Since (per the spec) updating the `is` attribute after creation should *not* cause an element to be upgraded to a custom element, what I did here was only look at the `is` attribute when the MutationObserver says that a node was *added*, and never look at the `is` attribute after that.

Since this polyfill has been rewritten to rely on monkeypatched methods instead of mutation observers, I'll have to figure out how to get the same functionality without the MutationObserver telling me if it is a newly added node or not.